lib/rspec/expectations/aggregated_failure.rb

```diff
@@ -0,0 +1,26 @@
+module RSpec
+  module Expectations
+    # Value object holding information about an expectation failure.
+    # Returned by FailureAggregator instead of raising an exception.
+    # @private
+    class AggregatedFailure < Object
+      class << self
+        attr_accessor :instance
+      end
+
+      private_class_method :new
+
+      def self.build(failure)
+        new(failure).freeze
+      end
+
+      def initialize(failure)
+        @failure = failure
+      end
+
+      def inspect
+        "AggregatedFailure: #{@failure}"
```

Comment:
I'd be ok with this being a static "singleton" instance within the other file that had an inspect of something like:
```
AggregatedFailure: This method caused a failure which has been supressed to be aggregated into our failure report by returning this value, further errors can be ignored.
```

but this doesn't need to be its own file or instance.,